app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/HorizontalSnapScrollViewCalculateDisplayColumnCountTest.kt

```diff
@@ -0,0 +1,215 @@
+                // Pixel 2 portrait (maxColumnCountForLayout=1)
+                pixel2Portrait(totalColumnCount = 1, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 2, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 3, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 4, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 5, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 6, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 7, calculatedColumnCount = 1),
+                pixel2Portrait(totalColumnCount = 55, calculatedColumnCount = 1),
+
+                // Pixel 2 landscape (maxColumnCountForLayout=4)
+                pixel2Landscape(totalColumnCount = 1, calculatedColumnCount = 1),
+                pixel2Landscape(totalColumnCount = 2, calculatedColumnCount = 2),
+                pixel2Landscape(totalColumnCount = 3, calculatedColumnCount = 3),
+                pixel2Landscape(totalColumnCount = 4, calculatedColumnCount = 4),
+                pixel2Landscape(totalColumnCount = 5, calculatedColumnCount = 4),
+                pixel2Landscape(totalColumnCount = 6, calculatedColumnCount = 4),
+                pixel2Landscape(totalColumnCount = 7, calculatedColumnCount = 4),
+                pixel2Landscape(totalColumnCount = 55, calculatedColumnCount = 4),
+
+                // Nexus 5 portrait (maxColumnCountForLayout=1)
+                nexus5Portrait(totalColumnCount = 1, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 2, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 3, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 4, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 5, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 6, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 7, calculatedColumnCount = 1),
+                nexus5Portrait(totalColumnCount = 55, calculatedColumnCount = 1),
+
+                // Nexus 5 landscape (maxColumnCountForLayout=4)
+                nexus5Landscape(totalColumnCount = 1, calculatedColumnCount = 1),
+                nexus5Landscape(totalColumnCount = 2, calculatedColumnCount = 2),
+                nexus5Landscape(totalColumnCount = 3, calculatedColumnCount = 3),
+                nexus5Landscape(totalColumnCount = 4, calculatedColumnCount = 4),
+                nexus5Landscape(totalColumnCount = 5, calculatedColumnCount = 4),
+                nexus5Landscape(totalColumnCount = 6, calculatedColumnCount = 4),
+                nexus5Landscape(totalColumnCount = 7, calculatedColumnCount = 4),
+                nexus5Landscape(totalColumnCount = 55, calculatedColumnCount = 4),
+
+                // Nexus 9 portrait, sw720dp (maxColumnCountForLayout=6)
```

Comment:
Should this be `maxColumnCountForLayout=4`?

Reply:
If I did not make a mistake here when writing it down it was `6` both in portrait and landscape mode.

Reply:
On the real device (Nexus 9) in portrait mode 4 rooms are shown, in landscape mode 6 rooms are shown.

Reply:
@cketti I checked the value on the device: it is `6` both for portrait and landscape. The calculation results in 4 columns because of the `160dip`.